### PR TITLE
update README

### PR DESCRIPTION
**Pull request by fivetran-kristin**
- Includes details on how to apply custom configurations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,50 @@ vars:
...
 
+### Changing the Build Schema (RECOMMENDED)
+By default this package will build all models in your <target_schema>.  This behavior can be tailored to your preference by making use of custom schemas. We highly recommend use of custom schemas for this package, as multiple sources are involved.  To do so, add the following configuration to your `dbt_project.yml` file: 
+
+```yml
+# dbt_project.yml
+
+...
+models:  
+  pinterest:
```
Comment (fivetran-joemarkiewicz) [inline README.md:121]: Should the `enabled: false` still be present for the Pinterest configs here?

**Review by fivetran-joemarkiewicz — APPROVED** [review]
The custom schema config looks good! I just had one question about the inclusion of the `enabled: false` being present in the pinterest model custom schema config. 

Unrelated, a customer pointed out that there was [no compatibility section](https://getdbt.slack.com/archives/C01D1R2JLLA/p1614935744050400?thread_ts=1614848349.030500&cid=C01D1R2JLLA) within the ReadMe indicating that the package is not compatible with Postgres. I know we stopped including the compatibility sections due to the Spark "coming soon" blurb. However, do you think we should add it back in to clarify this is only compatible with Redshift, Snowflake, and BigQuery?

File: README.md (hunk 1 above)
Comment (fivetran-kristin) [reply README.md:121]: good call, nope they shouldn't. I've removed.

**Comment by fivetran-kristin** [comment]
Thanks, Joe! I just updated the model configs and have also added our standard "Database Support" section.

**Pull request merged**